pkg/odo/cli/project/create.go

```diff
@@ -53,26 +53,37 @@ func (pco *ProjectCreateOptions) Validate() (err error) {
 
 // Run runs the project create command
 func (pco *ProjectCreateOptions) Run() (err error) {
+
+	successMessage := fmt.Sprintf(`Project '%s' is ready for use`, pco.projectName)
+
+	// Create the "spinner"
+	s := &log.Status{}
+
+	// If the --wait parameter has been passed, we add a spinner..
```

Comment:
Ah yup, this is normal. we create the "spinner" before the if statement so that we only have 1 spinner at a time / make sure that everything is correct.

Reply:
Thanks for explaining this. :smile: :+1: